python/ray/_private/services.py

```diff
@@ -1501,6 +1505,15 @@ def start_gcs_server(
         command += [f"--redis_username={redis_username}"]
     if redis_password:
         command += [f"--redis_password={redis_password}"]
+
+    devnull_handle = None
+    stdout_file = None
+    if ray_log_filepath:
+        devnull_handle = open(os.devnull, "w")
+        stdout_file = devnull_handle
```

Comment:
Should we set stdout_file to None in case gcs server uses `std::cout <<`?

Reply:
I'm not too worried about it, actually the current implementation only focuses on spdlog-oriented logging, I have left a comment on `std::cout` usage
https://github.com/ray-project/ray/blob/44e4cd6dd3681e757caaa0923ba7bcfaeab070ed/src/ray/gcs/gcs_server/gcs_server_main.cc#L52-L55

Reply:
If we really worry about the stdout usage, would it make sense to you if we address it in the next followup PR?